kernel/base/src/main/java/com/twosigma/beakerx/kernel/commands/MagicCommand.java

```diff
@@ -190,6 +204,36 @@ private MagicCommandFunctionality classpathAddJar() {
     };
   }
 
+  private void validateWildcardPath(String path) {
+    if (!path.matches("^([^*]*\\*){1}[^*]*$") || !path.endsWith("*")) {
```

Comment:
not sure what this does.
i believe all we want to say is if it has a star, it has to be at the end only.
matches is too complicated for just that.